nntrainer/src/databuffer.cpp

```diff
@@ -425,24 +425,37 @@ bool DataBuffer::getDataFromBuffer(
 
 int DataBuffer::setDataFile(std::string path, DataType type) {
   int status = ML_ERROR_NONE;
-  std::ifstream data_file(path);
-  if (!data_file.good()) {
-    ml_loge("Error: Cannot open configuraiotn file %s", path.c_str());
-    return ML_ERROR_INVALID_PARAMETER;
-  }
+  std::ifstream data_file(path.c_str());
 
   switch (type) {
   case DATA_TRAIN:
+    if (!data_file.good()) {
+      ml_loge(
+        "Error: Cannot open data file, Datafile is necessary for training");
+      return ML_ERROR_INVALID_PARAMETER;
+    }
     train_file = path;
     break;
   case DATA_VAL:
+    if (!data_file.good()) {
+      ml_logw("Waring: Cannot open validation data file. Cannot validate "
```

Comment:
I think it is typo `Waring` -> `Warning` and also for line 449

Reply:
Thanks.!